lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -425,6 +439,32 @@ class ReportUIFeatures {
         this._toggleDarkTheme();
         break;
       }
+      case 'swap-locale': {
+        if (!this._swapLocaleOptions) throw new Error('must call .initSwapLocale first');
+
+        // TODO: esmodules would be nice here...
```

Comment:
I figure we can gate this feature to just browsers supporting ESModules. The only complication is actually generating an esmodule bundle from commonjs code ... Should probably use rollup for this.

Reply:
not 100% sold the complexity of introducing rollup is lower than the complexity of 

```js
const script = document.createElement('script')
script.src = './i18n-bundle.js'
document.body.appendChild(script)
await new Promise(r => document.addEventListener('i18nloaded', r, {once: true})

// in bundle
document.dispatchEvent(new Event('i18nloaded'))
```

buuuuut if you want to use this as an opportunity to start transitioning some to rollup, I understand :)

Reply:
alright let's do it this way. using `import` will require, as you said, rollup investment, but also upgrading eslint (`import` keyword causes parse error right now) and I don't wanna update our linter right now.